packages/contracts/src/EmailAuth.sol

```diff
@@ -26,8 +26,9 @@ struct EmailAuthMsg {
 /// @dev Inherits from OwnableUpgradeable and UUPSUpgradeable for upgradeability and ownership management.
 contract EmailAuth is OwnableUpgradeable, UUPSUpgradeable {
     bytes32 public accountSalt;
-    ECDSAOwnedDKIMRegistry public dkim;
+    IDKIMRegistry public dkim;
     Verifier public verifier;
+    address module;
```

Comment:
Thought on naming convention:

With the current contract architecture, we have the recovery module(s) which are the 7579 modules that hold the logic to recover an account/validator (e.g. `OwnableValidatorRecoveryModule`, `SafeRecoveryModule`, or the potentially new generic `ValidatorRecoveryModule`), and we have `ZkEmailRecovery`, which is separate from these account modules.

With this in mind, I'm wondering if `module` is the correct term here, as then we are referring to two different types of contract as modules. Wdyt about this?

Not too opinionated on what other names could be but some potential options:
- `manager`
- `controller`